tests/testthat/test-register_logger.R

```diff
@@ -39,24 +39,23 @@ testthat::test_that("An additional logging namespace is created after register_l
 testthat::test_that("register_logger does not throw if passed a correct logger layout", {
   withr::with_options(
     new = list(teal.log_level = logger::INFO),
-    code = testthat::expect_error(register_logger(namespace = "test", layout = "{msg}"), NA)
+    code = testthat::expect_no_error(register_logger(namespace = "test", layout = "{msg}"))
   )
 })
 
 testthat::test_that("register_logger does not throw if passed a correct logger level", {
   withr::with_options(
     new = list(teal.log_layout = "{msg}"),
-    code = testthat::expect_error(register_logger(namespace = "test", level = logger::INFO), NA)
+    code = testthat::expect_no_error(register_logger(namespace = "test", level = logger::INFO))
   )
 })
 
 testthat::test_that("register_logger does not throw if passed NULL to either log_layout or log_level
```

Comment:
Linter errors, the code below should solve both

```r
testthat::test_that(
  "register_logger does not throw if passed NULL to either log_layout or log_level and valid options are set",
  {
    withr::with_options(
      new = list(teal.log_layout = "{msg}", teal.log_level = logger::INFO),
      code = testthat::expect_no_error(
        register_logger(namespace = "test", level = NULL, layout = NULL)
      )
    )
  }
)

testthat::test_that(
  paste(
    "register_logger",
    "does not throw if passed NULL to arguments, options are not set and the system variables are set to valid value"
  ), {
    withr::local_envvar(.new = list(TEAL.LOG_LAYOUT = "{msg}", TEAL.LOG_LEVEL = "INFO"))
    withr::local_options(.new = list(teal.log_layout = NULL, teal.log_level = NULL))
    testthat::expect_no_error(register_logger(namespace = "test", layout = NULL, level = NULL))
  }
)
```

Reply:
fixed in here -thanks cf9b27951cc3feba03be863492ad845511134354